CHANGELOG.md

```diff
@@ -4,6 +4,36 @@
 
 ### New Features
 
+### Breaking Changes
+
+### New Rules (0)
+
+-
+
+### Bug Fixes
+
+### capa explorer IDA Pro plugin
+
+### Development
+
+### Raw diffs
+- [capa v7.2.0...master](https://github.com/mandiant/capa/compare/v7.2.0...master)
+- [capa-rules v7.2.0...master](https://github.com/mandiant/capa-rules/compare/v7.2.0...master)
+
+### v7.2.0
+capa v7.2.0 adds support to analyze DRAKVUF sandbox dynamic analysis results. This release also introduces a first version of capa explorer web: a web-based user interface to inspect capa results using your browser. capa explorer web is available at https://mandiant.github.io/capa/explorer/#/.
```

Comment:
I think we should include links to more documentation about these new features. They are exciting! They deserve more than one sentence each, even if its not inline.

Reply:
I have updated this now, please feel free if you would like to reword it.